### PR TITLE
docs: translate how_to_release docs

### PR DESCRIPTION
**Pull request by yuluo-yx**

### CODE REVIEW

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/community/how_to_release.md [hunk 1]
```diff
@@ -0,0 +1,531 @@
...
+id: how_to_release
+---
+
+本文主要介绍如何发布新版本的 Apache Fury。
+
+## 介绍
+
+源代码发布是 Apache 最重视以及最重要的部分。
+
+请多注意许可证和发布得软件签名问题。发布软件是一件严肃的事情，并会产生相应的法律后果。
```
Comment (chaokunyang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/community/how_to_release.md:13]:
```suggestion
请注意许可证和发布的软件签名问题。发布软件是一件严肃的事情，并会产生相应的法律后果。
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/community/how_to_release.md [hunk 2]
```diff
@@ -0,0 +1,531 @@
...
+
+此发布过程在 Ubuntu 系统中运行，需要以下几个环境依赖：
+
+- JDK 1.8+
+- Apache Maven 3.x+
+- Python 3.8
+- GnuPG 2.x
+- Git
+- SVN（Apache 基金会使用 svn 来托管项目发布）
+- **设置环境变量**：如果您在不同的目录下配置了 gpg 密钥，请执行 `export GNUPGHOME=$(xxx)` 设置为系统环境变量。
```
Comment (chaokunyang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/community/how_to_release.md:27]:
```suggestion
- **设置环境变量**：如果您在不同的目录下配置了 gpg 密钥，请执行 `export GNUPGHOME=$(xxx)` 导出环境变量。
```

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/community/how_to_release.md [hunk 3]
```diff
@@ -0,0 +1,531 @@
...
+
+访问 https://dist.apache.org/repos/dist/dev/incubator/fury/ 以检查 artifacts 是否正确上传。
+
+### 如果出现问题该怎么办
+
+如果某些文件是意外出现或者发生某些错误，则需要删除相关内容并执行 `svn delete`，然后重复上述上传过程。
+
+## 投票
+
+作为一个孵化项目，Fury 需要 FUry 社区和孵化器社区的投票。
```
Comment (chaokunyang) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/community/how_to_release.md:248]:
```suggestion
作为一个孵化项目，新版本发布需要 Apache Fury 社区和孵化器社区的投票。
```

**Review by chaokunyang — APPROVED** [review]
Looks great

**Comment by chaokunyang** [comment]
Looks strange, it doesn't show chinese document:
image

**Comment by yuluo-yx** [comment]
image

@chaokunyang try refreshing?

**Comment by yuluo-yx** [comment]
image

Is it possible that this deploy ci is not completed and the site content is not updated?

**Pull request merged**